owncloudData/src/main/java/com/owncloud/android/data/storage/LocalStorageProvider.kt

```diff
@@ -86,6 +89,52 @@ sealed class LocalStorageProvider(private val rootFolderName: String) {
     @SuppressLint("UsableSpace")
     fun getUsableSpace(): Long = getPrimaryStorageDirectory().usableSpace
 
+    /**
+     * Checks if there is user data which does not have a corresponding account in the Account manager.
+     */
+    private fun getDanglingAccountDirs(context: Context?): List<File> {
+        val am = AccountManager.get(context)
```

Comment:
Do not save some letters 👀   we should try to name it `accountManager` so we know exactly what is going on 👍

Reply:
BTW, we have some AccountUtils to get logged accounts, we can reuse that code to retrieve the accounts.

Reply:
Sure. Should I use AccountUtils as a Singleton or should I use it via proper dependency injection?

Reply:
Also why is AccountUtils existing twice? Once [here](https://github.com/owncloud/android/blob/2a23807f4ea79f6e7fa2683af59242892993b690/owncloudApp/src/main/java/com/owncloud/android/authentication/AccountUtils.java) and once [here](https://github.com/owncloud/android-library/blob/351682cc7f5d91c1c6d21d7a8debd541d7c45674/owncloudComLibrary/src/main/java/com/owncloud/android/lib/common/accounts/AccountUtils.java#L115)